.github/workflows/pull-request.yml

```
@@ -107,13 +125,27 @@
+          python3 -m venv venv
+          source venv/bin/activate
           python -m pip install --upgrade pip
-          pip install -r requirements.txt
-      - name: Test with pytest
-        run: |
-          pytest --cov --junitxml=junit.xml
+          pip3 install -r requirements.txt
+          pip install pytest-cov
+          tests/bin/error_code_report.py
+          sudo systemctl start mysql.service
+          sudo mysql -u${{ env.DATABASE_ROOT_USERNAME}} -p${{ env.DATABASE_ROOT_PASSWORD }} -e 'CREATE DATABASE ${{ env.DATABASE_NAME }};' 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)